playbooks/roles/scorm/tasks/scorm_base.yml

```diff
@@ -0,0 +1,29 @@
+
+- name: FIXME create application user (copied over from edxapp/tasks/main.yml)
```

Comment:
thanks for reviewing and catching this,  TBH I don't fully recall now.   I'm going to check whether this is really needed.